graylog2-server/src/main/java/org/graylog/plugins/views/search/Search.java

```diff
@@ -30,6 +30,9 @@
 import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
 import org.graylog.plugins.views.search.views.PluginMetadataSummary;
+import org.graylog2.contentpacks.ContentPackable;
+import org.graylog2.contentpacks.EntityDescriptorIds;
+import org.graylog2.contentpacks.model.entities.SearchEntity;
```

Comment:
As discussed, all changes look great to me, except the fact that the `views` package now also depends on `contentpacks`. 
It would be better if `SearchEntity` would provide a factory method instead, e.g., `SearchEntity.from(Search search)`.

I understand that this would introduce an inconsistency with how alerting supports content packs, but from my point of view it would be preferrable to temporarily introduce an inconsistency between how alerting and views are dealt with in the content packs context over introducing a dependency from views to content packs.

So I would suggest cleaning it up for views in this pr and doing a follow up pr where the same is done for alerting.